ReactWindows/ReactNative/Modules/Internationalization/InternationalizationModule.cs

```diff
@@ -0,0 +1,65 @@
+﻿using ReactNative.Bridge;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace ReactNative.Modules.Internationalization
+{
+    /// <summary>
+    /// A module that allows JS to get/set right to left preferences.
+    /// </summary>
+    class InternationalizationModule : NativeModuleBase
```

Comment:
>InternationalizationModule [](start = 10, length = 26)

I sort of don't like that the name is so much different from the JS module name.  Is there a precedent for that in other modules?